charts/bitcoind/templates/networkpolicy-bitcoind.yaml

```diff
@@ -0,0 +1,13 @@
+kind: NetworkPolicy
```

Comment:
Should be renamed to `bitcoind-networkpolicy.yaml` to conform to the conventions seen with bitnami charts.